benchmark/README.md

```diff
@@ -59,23 +59,23 @@ All platforms use cuDnn-v5.1. You might see that caffe is slower, because the wo
 | TensorFlow   | 9     | 15       | 28      | 59       |
 | Caffe        | 9.373  | 16.6606  | 31.4797 | 59.719  |
 
-##### Notation
+**Notation**
 
-All the tests in caffe use `caffe time` to execute, which is not including the parameter updating process. But the time in PaddlePaddle and TensorFlow contains it.
+All the experiments in caffe use `caffe time` to execute, which does not include the time of parameter updating. The time in PaddlePaddle and TensorFlow contains it. But, compared with the total time, the time of parameter updating is relatively little.
```

Comment:
All the single-GPU experiments in caffe use `caffe time` to calculate the elapsed time, which does not include the parameter updating time. However, both PaddlePaddle and TensorFlow benchmark contain this parameter updating time. As compared with the total time, this part is relatively little, we can ignore it.